### DISCUSSION TITLE
GitHub Actions bug on page

### DISCUSSION THREAD

**Discussion by purehaTime** [community/community#43775] [Actions] [answered]
### Select Topic Area

Bug

### Body

Hi! I'm try to use github actions and after my new one commit on github action pages. I catch some strange bug with html layout on page.

I use Chrome latest version  and disable all extension on page.
Check out screenshots
<img width="1749" alt="Screenshot_2" src="https://user-images.githubusercontent.com/3066053/211407585-4de91778-b565-42dd-a28e-5a6735770c4d.png">
<img width="1724" alt="Screenshot_3" src="https://user-images.githubusercontent.com/3066053/211407589-c1fde1cd-9889-48d6-a999-823fe36c45f6.png">

also it is public project. I use grps generation via Visual Studio and make mistake of protobug file

**Comment by purehaTime** [comment]
link to repo: https://github.com/purehaTime/NinjaCollector

**Comment by qbunt** [comment, marked as answer]
Seeing this on every public/private project I can see

**Reply by purehaTime** [reply]
> Seeing this on every public/private project I can see

aIfter fix my protobuf (in my project) file it was gone

**Reply by qbunt** [reply]
not seeing it anymore, maybe you fixed it for everyone 🤣

**Reply by purehaTime** [reply]
I try to break again and its not reproduced
